tests/test_modeling_common.py

```diff
@@ -1463,6 +1463,197 @@ def test_pt_tf_model_equivalence(self):
 
         import transformers
 
+        def prepare_tf_inputs_from_pt_inputs(pt_inputs_dict):
+
+            tf_inputs_dict = {}
+            for key, tensor in pt_inputs_dict.items():
+                # skip key that does not exist in tf
+                if type(tensor) == bool:
+                    tf_inputs_dict[key] = tensor
+                elif key == "input_values":
+                    tf_inputs_dict[key] = tf.convert_to_tensor(tensor.cpu().numpy(), dtype=tf.float32)
+                elif key == "pixel_values":
+                    tf_inputs_dict[key] = tf.convert_to_tensor(tensor.cpu().numpy(), dtype=tf.float32)
+                elif key == "input_features":
+                    tf_inputs_dict[key] = tf.convert_to_tensor(tensor.cpu().numpy(), dtype=tf.float32)
+                # To deal with the edge cases from `TFTapasForQuestionAnswering`.
+                # PyTorch can deal with type casting automatically, but TensorFlow is more strict!
+                # TODO: find a clean/better way to deal with these extra keys that are not common.
+                elif key == "float_answer":
+                    tf_inputs_dict[key] = tf.convert_to_tensor(tensor.cpu().numpy(), dtype=tf.float32)
+                elif key == "numeric_values":
+                    tf_inputs_dict[key] = tf.convert_to_tensor(tensor.cpu().numpy(), dtype=tf.float32)
+                elif key == "numeric_values_scale":
+                    tf_inputs_dict[key] = tf.convert_to_tensor(tensor.cpu().numpy(), dtype=tf.float32)
```

Comment:
```suggestion
                    tf_inputs_dict[key] = tf.convert_to_tensor(tensor.cpu().numpy(), dtype=tf.float32)
```
Think it would be a bit more readable if we do:

```
elif key in ["float_answer", "numeric_values", ...]:
    tf_inputs_dict[key] = tf.convert_to_tensor(tensor.cpu().numpy(), dtype=tf.float32)
```
Think we collapse 6 elif statement into 1 here

Reply:
@patrickvonplaten the suggestion doesn't have any change. Do you want to renew the suggestion here?
Otherwise, I will collapse only the new added `float_answer, numeric_values, numeric_values_scale` here, but leave the main inputs untouched. Is this OK?

Reply:
made the change.